docs/For Users/Advanced/Use Native Node Modules.md

```diff
@@ -3,14 +3,22 @@
 
 [TOC]
 
+## npm and node-gyp
+
+!!! note "New for 0.13.0"
+    Starting from 0.13.0, native modules built by `node-gyp` or `npm` in upstream can be directly supported by NW.js.
+
 `npm` tool provided by Node.js builds the native modules, i.e. [C/C++ Addons](https://nodejs.org/api/addons.html), at the time of installation.
 
-Starting from 0.13.0, native modules built by node-gyp or npm in upstream can be supported.
+On Linux and OSX, the native module built with `npm` or `node-gyp` can be loaded by NW.js directly.
+
+On Windows you'll need to replace the file
+`<npm-path>\node_modules\node-gyp\src\win_delay_load_hook.c` with the one at https://github.com/nwjs/nw.js/blob/nw13/tools/win_delay_load_hook.c before using `npm` or `node-gyp`. Typically `<npm-path>` is located in the installation directory of Node.js, such as `C:\Program Files\nodejs\node_modules\npm` if Node.js is installed with MSI installer.
 
-In Linux and OSX you can just load the native module directly. In windows you'll need to replace the file
-%APPDATA%\npm\node_modules\node-gyp\src\win_delay_load_hook.c with the one at https://github.com/nwjs/nw.js/blob/nw13/tools/win_delay_load_hook.c 
```

Comment:
should we keep this info on APPDATA dir somehow? It would be useful. At least it applies to me.